File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/OpenPopupAction.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class OpenPopupAction{

	@SerializedName("popupType")
	private String popupType;

	@SerializedName("popup")
	private Popup popup;

	public void setPopupType(String popupType){
		this.popupType = popupType;
	}

	public String getPopupType(){
		return popupType;
	}

	public void setPopup(Popup popup){
		this.popup = popup;
	}

	public Popup getPopup(){
		return popup;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/OpenPopupAction.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class OpenPopupAction : Parcelable, Serializable {
    @SerializedName("popupType")
    var popupType: String? = null

    @SerializedName("popup")
    var popup: Popup? = null
    override fun toString(): String {
        return "OpenPopupAction{" +
                "popupType = '" + popupType + '\'' +
                ",popup = '" + popup + '\'' +
                "}"
    }
}